xAPI.md

```diff
@@ -507,6 +507,45 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 ## 7.0 xAPI Components
 
+This section explains and shows graphically how different pieces of xAPI can fit together.  
+
+##### Tracking Experiences Through Systems
+
+![Data Flow in xAPI](/xAPIDataFlow.jpg)
+
+Figure 1 shows the tracking of experiences through the data flow of a series of systems.  The cloud in the picture 
+represents a Learning Management System, or LMS.  An LMS, if present, would likely encapsulate everything within 
+the cloud.  In either case, a user will be dependent on a relationship between content and one or more Learning 
+Record Stores.  It is also possible a learner is dependent on an Activity Provider to deliver and/or authorize use 
+of content to them.  Once this triangle of authorization and access is complete, a registration exists for that 
+user in that content for that LRS.  LRSs, at request of the user, can also share data to other LRSs that may or 
+may not be a part of the LMS framework.  Web Services, again as a part of or independent of the LMS, can pull 
+data from one or more LRSs, send them through other Web Services or LRSs.
+
+##### Activity Data and Metadata
+
+![xAPI Activity Data and Metadata](/xAPIMetadata.jpg)
+
+Understanding how a single activity (as uniquely identified by its IRI) is produced, populated, updated, and 
+consumed is a key concept in xAPI. Figure 2 shows this process.  xAPI Statements with the Object as an Activity, 
+have metadata within its Activity Definition.  For each Statement, this amount may be large or extremely sparse.  All Statements are sent via the Statement endpoint to the LRS.  Within the Activity is an IRI which also could have metadata located at it (note that this is different from the moreinfo URL in the Activity Definition, which could also have information located at it, is not involved in any other way in this information flow.  The base portion of the IRI as well as any metadata where the IRI resolves is under the control of the Metadata Provider.  Any metadata at the IRI is the authoritative source of metadata, and will populate the overall Activity Definition. This is known as the canonical definition of the activity can be returned on its own by querying 'canonical' on the Statement Endpoint.  It can then be used by the Metadata Consumer.  Other forms of the metadata can be returned via other query options of the Statement Endpoint or the Activity Definition in its entirety can be obtained via the Activities Endpoint.  The LRS is responsible for population and updates of this Activity Definition as well as sending the corresponding metadata upon being queried by the Statement Resource (Endpoint).
+
+
+##### Agent and Profile Management
+
+![xAPI Activity Data and Metadata](/xAPIAgentsAndEndpoints.jpg)
```

Comment:
* Suggest using a collection of fake web service logos instead of the device images to represent platforms. 
* I think this diagram and section should just be about the persona/person concept. Including documents and statements makes it more complicated.